### PR TITLE
Plex Mod: Audnexus.bundle

### PR DESCRIPTION
**Pull request by tylernguyen**

### CODE REVIEW

File: root/etc/s6-overlay/s6-rc.d/init-mod-plex-audnexus-add-package/up
Comment (aptalca) [inline root/etc/s6-overlay/s6-rc.d/init-mod-plex-audnexus-add-package/up:1]: This file's content needs to point to the correct run file

Comment (tylernguyen) [reply root/etc/s6-overlay/s6-rc.d/init-mod-plex-audnexus-add-package/up:1]: Fixed.

**Comment by aptalca** [comment]
Please test the built PR mod and confirm it works. Then we'll merge.

Thanks

**Comment by tylernguyen** [comment]
Tested. I can confirm that it works.

Please merge. Thank you

**Review by aptalca — APPROVED** [review]

**Pull request merged**